cores/migrations/20240628024728_AddSummaryToPageSnapShot.up.sql

```diff
@@ -1,2 +0,0 @@
--- Add migration script here
-ALTER TABLE PageSnapshot ADD COLUMN summary TEXT DEFAULT '' NOT NULL;
```

Comment:
removed `Summary` Column from `PageSnapshot` table and delete the `.up` && `.down`